tidb-lightning/tidb-lightning-physical-import-mode-usage.md

```diff
@@ -31,12 +31,13 @@ data-source-dir = "/data/my_database"
 
 [conflict]
 # Starting from v7.3.0, a new version of strategy is introduced to handle conflicting data. The default value is "". Starting from v8.0.0, TiDB Lightning optimizes the conflict strategy for both physical and logical import modes.
-# - "": TiDB Lightning does not detect or handle conflicting data. If the source file contains conflicting primary or unique key records, the subsequent step reports an error.
+# - "": in the physical import mode, TiDB Lightning does not detect or handle conflicting data. If the source file contains conflicting primary or unique key records, the subsequent step reports an error. In the logical import mode, TiDB Lightning converts the "" strategy to the "error" strategy for processing.
```

Comment:
This doc is for physical import mode. Perhaps, it is unnecessary to mention the handling way of logical import mode in the above sentence?

Reply:
modified